### PR TITLE
[Parameters] Drop operator selector from filter popover

### PR DESCRIPTION
**Pull request by uladzimirdev**
Closes https://github.com/metabase/metabase/issues/41313

### Description

When you're editing a dashboard and when you want to add a filter, there used to be 2 clicks: type of the filter and operator. With this PR we guess the default operator for a filter and keep only 1 click to create a filter.

Check commits to better understand the diff

### Demo

https://github.com/metabase/metabase/assets/125459446/ec7704e4-bda5-48ee-ad38-30ad5e21609d

### CODE REVIEW

File: frontend/src/metabase/parameters/utils/dashboard-options.js [hunk 1]
```diff
@@ -56,19 +56,19 @@ export function getDefaultOptionForParameterSection() {
       }
 
       if (sectionId === "location") {
-        defaultOption = options.find(o => o.name === "Is");
+        defaultOption = options.find(o => o.type === "string/=");
```
Comment (uladzimirdev) [inline frontend/src/metabase/parameters/utils/dashboard-options.js:59]: we need to support other languages and thanks god we have tests for it

File: frontend/src/metabase/dashboard/components/ParametersPopover.jsx [hunk 2]
```diff
@@ -28,39 +33,26 @@ export class ParametersPopover extends Component {
...
-      const parameterSection = _.findWhere(this.PARAMETER_SECTIONS, {
-        id: section,
-      });
-      return (
-        <ParameterOptionsPane
-          options={parameterSection && parameterSection.options}
-          onSelectOption={option => {
-            onAddParameter(option);
+
+    return (
```
Comment (uladzimirdev) [inline frontend/src/metabase/dashboard/components/ParametersPopover.jsx:38]: main change is here

File: frontend/src/metabase/parameters/utils/dashboard-options.js (hunk 1 above)
Comment (kamilmielnik) [reply frontend/src/metabase/parameters/utils/dashboard-options.js:59]: Oh wow :man_facepalming:

File: e2e/support/helpers/e2e-dashboard-helpers.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { visitDashboard } from "./e2e-misc-helpers";
-import { menu, popover } from "./e2e-ui-elements-helpers";
+import { menu, popover, sidebar } from "./e2e-ui-elements-helpers";
```
Comment (kamilmielnik) [inline e2e/support/helpers/e2e-dashboard-helpers.js:2]: The value depends on the operator. If we're changing operator from multiple-value one to a single-value one, the default value still mentions 2 values

https://github.com/metabase/metabase/assets/6830683/a0aff875-d2ee-46c1-8eb1-14fb23b28114

Comment (kamilmielnik) [reply e2e/support/helpers/e2e-dashboard-helpers.js:2]: Similarly, when changing the filter type, the value can get "broken"

https://github.com/metabase/metabase/assets/6830683/c620e55e-f32d-44ea-a558-e06af02d1d43

Comment (kamilmielnik) [reply e2e/support/helpers/e2e-dashboard-helpers.js:2]: I think the simplest solution would be to clear the value if it cannot be used after changing filter type or the operator.

Comment (kamilmielnik) [reply e2e/support/helpers/e2e-dashboard-helpers.js:2]: Oh, now I see you covered changing the filter type in #41486.
But we still should cover changing the operator type.

Comment (uladzimirdev) [reply e2e/support/helpers/e2e-dashboard-helpers.js:2]: you're right, there should be some additional logic to understand if default value can be re-used. but I think the simplest solution for now is to drop default value in both cases

**Review by kamilmielnik — APPROVED** [review]
:+1:

**Pull request merged**